src/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py

```diff
@@ -1290,23 +1360,36 @@ def write_sorting(
                 nwbfile=nwbfile,
                 property_descriptions=property_descriptions,
                 skip_properties=skip_properties,
-                skip_features=skip_features,
                 use_times=use_times,
                 write_as=write_as,
                 units_name=units_name,
                 units_description=units_description,
             )
+            add_wave_forms_to_units_table(
+                sorting=sorting,
+                nwbfile=nwbfile,
+                skip_features=skip_features,
+                units_table_name=units_table_name,
+                unit_table_description=units_table_description,
+                write_units_table_in_processing_module=write_units_table_in_processing_module,
```

Comment:
I think it's fine/readable to remove the above lines and do the kwarg aliasing here

```python
    units_table_name = units_name
    units_table_description = units_description
```